CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [1.137.2] - 2024-04-05
+
+### Features / Changes
+
+- Show reactions in summaries ([#5387](https://github.com/deltachat/deltachat-core-rust/pull/5387)).
+
+### Fixes
+
+- Do not show empty summary if message reacted to is deleted.
+- Do not ignore `Message::load_from_db` errors.
+
+### Miscellaneous Tasks
+
+- cargo: Bump rustyline from 13.0.0 to 14.0.0.
+- Update chrono from 0.4.34 to 0.4.37.
+- Update from brotli 3.4.0 to brotli 4.0.0.
+- Upgrade `h2` from 0.4.3 to 0.4.4.
+- Upgrade `image` from 0.24.9 to 0.25.1.
+- cargo: Bump fast-socks5 from 0.9.5 to 0.9.6.
+
+### Other
+
+- Merge pull request #5404 from deltachat/dependabot/cargo/rustyline-14.0.0
+- Update node constants for #5387 ([#5429](https://github.com/deltachat/deltachat-core-rust/pull/5429))
```

Comment:
same here, do we need to mention a fix for sth that was not broken in the last version?